extension/constants.js

```diff
@@ -90,7 +94,9 @@ constants.SurveyLocation = {
   PHISHING_PROCEED: 'phishing-proceed.js',
   PHISHING_NOPROCEED: 'phishing-noproceed.js',
   EXTENSION_PROCEED: 'extension-proceed.js',
-  EXTENSION_NOPROCEED: 'extension-noproceed.js'
+  EXTENSION_NOPROCEED: 'extension-noproceed.js',
+  HTTPS: 'httpwithans.js',
```

Comment:
Hahahahahaha. Changed it to visited-http.js and visited-https.js, although I am sad to lose my story about a goatherd in a process.